src/SFA.DAS.ApplyService.Web/wwwroot/stylesheets/application.css

```diff
@@ -1303,4 +1308,16 @@
     line-height: 1.25;
     display: block;
     margin-bottom: 15px;
-}
\ No newline at end of file
+}
+
+.status--fail {
+    padding-left: 15px;
+    border-left: 5px solid #d4351c;
+    margin-bottom: 20px;
+}
+
+.status--pass {
+    padding-left: 15px;
+    border-left: 5px solid #00703c;
+    margin-bottom: 20px;
+} 
```

Comment:
These look familiar. Are they different styling to the custom tags used elsewhere?

Reply:
yes